src/Stripe.net/Infrastructure/Middleware/ParserPlugin/PaymentIntentAllowedSourceTypesPlugin.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Newtonsoft.Json;
+
+namespace Stripe.Infrastructure.Middleware
+{
+    internal class PaymentIntentAllowedSourceTypesPlugin : IParserPlugin
```

Comment:
This one is dirty/sad but it's just temporary because I needed to make it work. We should ship only after @ob-stripe other PR https://github.com/stripe/stripe-dotnet/pull/1218 instead.